.travis.yml

```diff
@@ -0,0 +1,67 @@
+#Gives permission to install SFDX
+sudo: true
+
+before_install:
+#
+# The before_install step will download the SFDX CLI and install it
+#
+
+#Decrypt server key
+- openssl aes-256-cbc -K $encrypted_b105d3438aa4_key -iv $encrypted_b105d3438aa4_iv
+  -in assets/server.key.enc -out assets/server.key -d
+#Assign variables
+- export CLIURL=https://developer.salesforce.com/media/salesforce-cli/sfdx-linux-amd64.tar.xz  
+- export SFDX_AUTOUPDATE_DISABLE=false
+- export SFDX_USE_GENERIC_UNIX_KEYCHAIN=true
+- export SFDX_DOMAIN_RETRY=300
+- export SFDX_DISABLE_APP_HUB=true
+- export SFDX_LOG_LEVEL=DEBUG
+#Create SFDX directory
+- mkdir sfdx
+#Install SFDX
+- wget -qO- $CLIURL | tar xJ -C sfdx --strip-components 1
+- "./sfdx/install"
+- export PATH=./sfdx/$(pwd):$PATH
+
+before_script:
+#
+# The before_script step initializes varibles used in the script step and authenticates to the devhub using JWT grant
```

Comment:
The before_script step initializes variables used in the script step and authenticates to the Dev Hub using JWT-based flow